utils/test_memcheck.supp

```diff
@@ -336,3 +336,45 @@
    fun:compat_call.constprop.0
    fun:internal_getgrouplist
 }
+{
+   spdk_env_init leak
+   Memcheck:Leak
+   ...
+   fun:spdk_env_init
+}
+{
+   spdk_notify_type_register leak
+   Memcheck:Leak
+   ...
+   fun:spdk_notify_type_register
+}
+{
+   bdev_user_io_getevents invalid read
```

Comment:
you've verified this one is safe?  Invalid read means outside of known range of memory (so in normal case, it would often mean a SEGV

Reply:
If that's the case then we probably warn to mark whatever memory segment this is in as valid rather than suppressing read errors from it.